index.html

```diff
@@ -157,7 +157,7 @@ <h2 class="hint-value">Around The House</h2>
     </div>
     <div class="solve-popup">
       <h2>Enter your guess:</h2>
-      <input class="solve-input" type="text" name="">
+      <input class="solve-input" type="text" name="" autofocus>
```

Comment:
what is autofocus?

Reply:
It puts the cursor right into the input when we click on the solve button